### PR TITLE
kcqrs: command handlers return response

### PR DESCRIPTION
**Pull request by mlesikov**
Sometimes, the command handlers should return response, such a
situations are when the http CRUD handlers requires a result to be
return to the client. This result usually contains the data of the view,
with some calculations and logic that were applied over it in the
aggregate

### CODE REVIEW

File: kcqrs-example/src/main/kotlin/com/clouway/kcqrs/example/commands/RegisterProductCommandHandler.kt [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+import com.clouway.kcqrs.core.CommandHandler
+import com.clouway.kcqrs.core.Response
 import com.clouway.kcqrs.example.domain.Product
 
 
-class RegisterProductCommandHandler(private val eventRepository: AggregateRepository) : CommandHandler<RegisterProductCommand> {
+class RegisterProductCommandHandler(private val eventRepository: AggregateRepository) : CommandHandler<RegisterProductCommand, String> {
 
-    override fun handle(command: RegisterProductCommand) {
+    override fun handle(command: RegisterProductCommand):Response<String> {
```
Comment (mgenov) [inline kcqrs-example/src/main/kotlin/com/clouway/kcqrs/example/commands/RegisterProductCommandHandler.kt]: looks not well-formatted.

**Comment by georgievJon** [comment]
This change should bump version to v0.0.7-SNAPSHOT(gradle.properties)

File: kcqrs-core/src/main/kotlin/com/clouway/kcqrs/core/Command.kt [hunk 2]
```diff
@@ -6,4 +6,4 @@ package com.clouway.kcqrs.core
  * 
  * @author Miroslav Genov (miroslav.genov@clouway.com)
  */
-interface Command
\ No newline at end of file
+interface Command<T>
```
Comment (georgievJon) [inline kcqrs-core/src/main/kotlin/com/clouway/kcqrs/core/Command.kt]: Suggest: 
If is introduced new interface will have compatibility with previous versions and will be possible smooth migration.

File: kcqrs-example/src/main/kotlin/com/clouway/kcqrs/example/commands/RegisterProductCommandHandler.kt (hunk 1 above)
Comment (mlesikov) [reply kcqrs-example/src/main/kotlin/com/clouway/kcqrs/example/commands/RegisterProductCommandHandler.kt]: done

File: kcqrs-core/src/main/kotlin/com/clouway/kcqrs/core/Command.kt (hunk 2 above)
Comment (mlesikov) [reply kcqrs-core/src/main/kotlin/com/clouway/kcqrs/core/Command.kt]: We will force a general version v1.0.0 as @mgenov suggests

**Comment by mlesikov** [comment]
@mgenov  can you take a look again, please

**Review by mgenov — APPROVED** [review]
LGTM!

Please make sure that all tests are passing as the CI is failing at the moment.

**Review by georgievJon — APPROVED** [review]

**Pull request merged**